pkg/apis/security/register.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package security contains experimental security policy API definitions.
```

Comment:
Add a llink to the proposal maybe?

Reply:
I think I'd rather avoid adding links to the design as folks without the right permissions (e.g., not belonging to the knative org) might be looking at this code ... I think with the easy way of linking the code to the PR, and having the design link in the PR, it should be enough... 
But if you feel we should add it, then go for it. No strong opinions.

Reply:
I don’t care deeply either way, but for anyone who can’t read the design it’s probably not useful anyway.

Reply:
I don’t care deeply either way, but for anyone who can’t read the design it’s probably not useful anyway.

Reply:
I like Nacho's approach. Thanks for the input.